lexicons/com/atproto/repo/checkMissingBlobs.json

```diff
@@ -0,0 +1,36 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.repo.listMissingBlobs",
```

Comment:
After seeing this, I could imagine moving migration-specific endpoints like this into their own segment, e.g. `com.atproto.migrate.listMissingBlobs`.

Reply:
I actually think this is a useful route in it's own right fwiw. There are some repos now that are missing blobs and this could be useful to figure out what

Reply:
does this let external folks discover which blobs have been purged for an account? or do purged blobs not show up in this output?

Reply:
(dholms replied below; I think the reason the github reply box didn't show in my review is that this comment was on the same line that divy commented on previously?)